pkgs/applications/video/vlc/default.nix

```diff
@@ -11,6 +11,7 @@
 , onlyLibVLC ? false
 , withQt5 ? true, qtbase ? null, qtsvg ? null, qtx11extras ? null, wrapQtAppsHook ? null
 , jackSupport ? false
+, skins2Support ? true, freetype ? null
```

Comment:
```suggestion
, skins2Support ? true, freetype, libXpm, freetype, libXext, libXinerama
```

Reply:
In this case I was doing likke the withQt5 example

If the package for some reason doesn't exist no problem if that flag is turned off.

Reply:
When using optional(s) the package can be null or anything and evaluation does not fail. The example is wrong here.

Reply:
So delete all the "? null" here?

Reply:
Yes